### PR TITLE
fakegps: added module to fake GPS input

### PR DESCRIPTION
**Pull request by tridge**
useful for indoor testing

### CODE REVIEW

File: MAVProxy/modules/mavproxy_fakegps.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+'''fake GPS input using GPS_INPUT packet'''
+
+import time
+from pymavlink import mavutil
+from MAVProxy.modules.lib import mp_module
+from MAVProxy.modules.lib import mp_settings
+
+class FakeGPSModule(mp_module.MPModule):
+    def __init__(self, mpstate):
+        super(FakeGPSModule, self).__init__(mpstate, "relay")
```
Comment (peterbarker) [inline MAVProxy/modules/mavproxy_fakegps.py]: Not a relay module.

File: MAVProxy/modules/mavproxy_fakegps.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        self.last_send = time.time()
+        self.FakeGPS_settings = mp_settings.MPSettings([("nsats", int, 16),
+                                                     ("lat", float, -35.363261),
```
Comment (peterbarker) [inline MAVProxy/modules/mavproxy_fakegps.py]: Indenting issues.   `flake8`s static analysis is worth keeping things flake8-clean IMO.

File: MAVProxy/modules/mavproxy_fakegps.py (hunk 1 above)
Comment (tridge) [reply MAVProxy/modules/mavproxy_fakegps.py]: fixed

File: MAVProxy/modules/mavproxy_fakegps.py (hunk 2 above)
Comment (tridge) [reply MAVProxy/modules/mavproxy_fakegps.py]: fixed

**Pull request merged**